PyAlbany/README.md

```diff
@@ -0,0 +1,93 @@
+# PyAlbany
+
+PyAlbany is a Python wrapper for Albany.
+
+### Prerequisites
+
+PyAlbany requires additional required dependencies:
+* A python executable (either Python 2 or 3),
+* Two python packages: numpy and mpi4py,
+* Swig 3.0.11 or higher (PyAlbany has been tested with swig 3.0.11 and 3.0.12).
+
+### Configuration
+
+In order to configure PyAlbany, both Trilinos and Albany have to be configured accordingly.
+
+PyAlbany can be configured with either Python 2 (tested with 2.7) or Python 3 (tested with 3.6).
+To switch from one python version to the other requires to recompile both Trilinos and Albany.
+
+##### Trilinos configuration
+PyAlbany adds one more Trilinos dependency to the already existing Albany dependencies: PyTrilinos.
+PyTrilinos is a Python wrapper of some of the capabilities of Trilinos packages.
+
+To enable PyAlbany, it is required to first build Trilinos with PyTrilinos enabled.
+To do so, Trilinos must be configured with the following options:
+```
+-D Trilinos_ENABLE_PyTrilinos:BOOL=ON \
+-D PyTrilinos_DOCSTRINGS:BOOL=OFF \
+```
+The first option enable PyTrilinos and the second one disables PyTrilinos docstrings. This last option is required if you are using Doxygen 1.8.13.
+
+While testing, it has been observed that setting the `MPI_BASE_DIR` can help while using PyTrilinos.
```

Comment:
I had to set `MPI_BASE_DIR` in order to compile, not iny for testing,

Reply:
Same as above.